2025-01-20 22:04:29,011 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:04:29,780 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:04:30,671 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:04:30,673 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:04:30,682 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:04:30,694 INFO: 127.0.0.1 - - [20/Jan/2025 22:04:30] "GET /socket.io/?EIO=4&transport=polling&t=PI5Y04e HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:04:30,703 INFO: 127.0.0.1 - - [20/Jan/2025 22:04:30] "POST /socket.io/?EIO=4&transport=polling&t=PI5Y07g&sid=DuUhGzmilFByn2Y-AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:04:30,706 INFO: 127.0.0.1 - - [20/Jan/2025 22:04:30] "GET /socket.io/?EIO=4&transport=polling&t=PI5Y07i&sid=DuUhGzmilFByn2Y-AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:00,738 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:00,927 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:01,450 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:07:02,216 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:07:02,217 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:02,224 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:02,232 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:02] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yb3o HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:02,263 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:02] "POST /socket.io/?EIO=4&transport=polling&t=PI5Yb7V&sid=5NIjtQ_5ZZb9xoW3AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:02,268 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:02] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yb7m&sid=5NIjtQ_5ZZb9xoW3AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:23,511 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/config/config.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:23,732 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:24,371 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:07:25,214 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:07:25,215 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:25,222 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:25,232 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:25] "GET /socket.io/?EIO=4&transport=polling&t=PI5Ygd4 HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:25,248 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:25] "POST /socket.io/?EIO=4&transport=polling&t=PI5Ygks&sid=TEJ7I8kZTgATYnA9AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:25,253 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:25] "GET /socket.io/?EIO=4&transport=polling&t=PI5Ygkw&sid=TEJ7I8kZTgATYnA9AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:33,334 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/config/production.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:33,534 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:33,980 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:07:34,651 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:07:34,652 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:34,659 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:34,831 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:34] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yj4W HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:34,853 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:34] "POST /socket.io/?EIO=4&transport=polling&t=PI5Yj4s&sid=80ZIX-bQsT84jGblAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:34,855 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:34] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yj4x&sid=80ZIX-bQsT84jGblAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:51,883 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:52,266 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:52,805 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:07:53,717 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:07:53,718 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:53,727 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:53,739 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:53] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yndp HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:53,748 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:53] "POST /socket.io/?EIO=4&transport=polling&t=PI5YniF&sid=Lo3lk05CanMlN3elAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:53,750 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:53] "GET /socket.io/?EIO=4&transport=polling&t=PI5YniI&sid=Lo3lk05CanMlN3elAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:07:53,754 INFO: 127.0.0.1 - - [20/Jan/2025 22:07:53] "GET /socket.io/?EIO=4&transport=polling&t=PI5YniO&sid=Lo3lk05CanMlN3elAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:01,843 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:02,072 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:02,521 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:08:03,177 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:08:03,178 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:03,187 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:03,197 INFO: 127.0.0.1 - - [20/Jan/2025 22:08:03] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yp-6 HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:03,218 INFO: 127.0.0.1 - - [20/Jan/2025 22:08:03] "POST /socket.io/?EIO=4&transport=polling&t=PI5Yq0E&sid=K12R388kAQD8p14FAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:03,219 INFO: 127.0.0.1 - - [20/Jan/2025 22:08:03] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yq0F&sid=K12R388kAQD8p14FAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:03,224 INFO: 127.0.0.1 - - [20/Jan/2025 22:08:03] "GET /socket.io/?EIO=4&transport=polling&t=PI5Yq0L&sid=K12R388kAQD8p14FAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:58,869 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:08:59,172 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
